.circleci/config.yml

```diff
@@ -112,10 +112,21 @@ jobs:
       - image: node:latest
     steps:
       - checkout
+      - jq/install
       - run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
         # if you try to use yarn here, it will attempt to use the wrong registry and throw 401s
       - run: npm run promote-dist-tags
 
+  promote-verify:
+    docker:
+      - image: node:latest
```

Comment:
@mshanemc should this be lts?

Reply:
Why?  Is there something from the (fairly limited) verification scripts that would fail?
[I have no objection to changing it, just curious about the reasoning]

Reply:
@mshanemc Given the issues we have seen recently with unanswered questions regarding v16 of node, I thought I would just raise the question.

Reply:
most of our build scripts are running on latest, not lts, except for the packing (where the bundled node version was defaulting to the build-time environment's version).